### DISCUSSION TITLE
Does Podman rootless deployment require ROCKET_PORT to be declared?

### DISCUSSION THREAD

**Discussion by JHBoricua** [dani-garcia/vaultwarden#2174] [General]
I'm curious why the Podman non-root example in the 'Starting a Container' section of the wiki is passing the ROCKET_PORT environmental variable along with the port mappings. Is this truly required? I was able to deploy a rootless vaultwarden container just fine with only the port mapping declared on my Fedora 35 Server. Just wondering if there was a specific reason for it as there's no mention about it in the wiki.

**Comment by BlackDex** [comment]
Well, for one, it is an example. And the default port configured within the container is 80, not 8080. So, if you want or need the port to run on something else then 80, you need that variable. Else you can remove that and use the default, and just put all other variables which you want to be changed in there

**Comment by JHBoricua** [comment]
Shoot, I just remembered something about Podman rootless. By default, you cannot map privileged ports (<1024) to a rootless container unless you add `net.ipv4.ip_unprivileged_port_start=80` to the /etc/sysctl.conf file. I totally forgot I did that earlier in my Podman setup. In that context the example makes complete sense. Would that be worth mentioning in the wiki?

**Reply by BlackDex** [reply]
Sure any extra useful information on reasons why settings are needed in some situations is good. So please go ahead and and this information.
